transform/macros/repo_events_by_period.sql

```diff
@@ -0,0 +1,11 @@
+{% macro repo_event_by_period(event, period) -%} 
+    SELECT
+        repo_id, 
+        repo_name, 
+        DATE_TRUNC('{{ period }}', event_date) AS date_month, 
+        COUNT(*) AS star_count,
```

Comment:
Instead of `star_count` we hsould use the event variable here to account for all types of event.